OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -108,16 +124,16 @@ public class TerrainInfo
 		{
 			public static readonly TerrainInfo Impassable = new TerrainInfo();
 
-			public readonly int Cost;
+			public readonly short Cost;
```

Comment:
It would be helpful for review and the git history / blame if the Cost size change could be split into its own commit.